moveit_core/exceptions/src/exceptions.cpp

```diff
@@ -35,14 +35,20 @@
 /* Author: Acorn Pooley, Ioan Sucan */
 
 #include <moveit/exceptions/exceptions.h>
-#include <ros/console.h>
+// #include <ros/console.h>
+#include <rcutils/logging_macros.h>
+#include "rclcpp/rclcpp.hpp"
+
+#define ROS_ERROR RCUTILS_LOG_ERROR
 
 moveit::ConstructException::ConstructException(const std::string& what_arg) : std::runtime_error(what_arg)
 {
-  ROS_ERROR_NAMED("exceptions", "Error during construction of object: %s\nException thrown.", what_arg.c_str());
+  // ROS_ERROR_NAMED("exceptions", "Error during construction of object: %s\nException thrown.", what_arg.c_str());
```

Comment:
remove dead code

Reply:
+1 to keeping NAMED

Reply:
You should use `RCLCPP_ERROR(some->way->to_get->node->get_logger(), "...");`, but if access to the node is impossible here, then something like `RCLCPP_ERROR(rclcpp::get_logger("moveit"), "...");`.

Reply:
Note, if you're going for the latter you can store the logger object somewhere to avoid creating a `std::string` each time you get logger.